ui.go

```diff
@@ -405,15 +407,14 @@ func connectView(ctx *ntcontext, w *nucular.Window) {
 func capabilitiesView(ctx *ntcontext, w *nucular.Window) {
 	w.Row(15).Dynamic(1)
 	w.Label("This program does not have the capabilities to function properly.", "CB")
-	w.Row(15).Dynamic(1)
-	w.Label("We require CAP_SYS_RESOURCE. If that doesn't mean anything to you, don't worry. I'll fix it for you.", "CB")
+	w.Row(100).Dynamic(1)
+	w.LabelWrap("We require CAP_SYS_RESOURCE. If that doesn't mean anything to you, don't worry. I'll fix it for you.")
 	if ctx.capsMismatch {
 		w.Row(15).Dynamic(1)
 		w.LabelColored("Warning: File has CAP_SYS_RESOURCE but our process doesn't.", "CB", orange)
 		w.Row(15).Dynamic(1)
 		w.LabelColored("Check if your filesystem has nosuid set or check the troubleshooting page.", "CB", orange)
 	}
```

Comment:
Yes, it changes the appearance when the window width is less than the text width. Instead of the text getting cut off, it now wraps.

When `w.Label` was used, `"CB"` defined the text alignment, with the `C` meaning centered and the `B` meaning the bottom of the label.

`w.LabelWrap` doesn't support custom alignment. Text will be aligned to the top-left, because it makes implementing wrapping much easier. So it would be equivalent to `"LT"` instead of `"CB"` when the text fits within the window.

Also see my latest comment for screenshots.